src/types/oicp/OICPAuthentication.ts

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable linebreak-style */
```

Comment:
Please do not disable linter rule at a file level. Fix the code instead.

Reply:
ok